pkgs/development/python-modules/serverlessrepo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, boto3
+, pyyaml
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "severlessrepo";
+  version = "0.1.8";
+
+  src = fetchPypi {
+    pname = "serverlessrepo";
```

Comment:
```suggestion
    inherit pname version;
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/commit/0d1ec03d1a3e70a546183e53795398ecfdbfd881#diff-09d026033975cc29501d417399d1fd94R16